packages/components/pie-input/README.md

```diff
@@ -87,6 +87,8 @@ import { PieInput } from '@justeattakeaway/pie-input/dist/react';
 | `readonly` | `boolean` | - | When true, the user cannot edit the control. Not the same as disabled. See [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/readonly) for more information. |
 | `type` | `'text'`, `'number'`, `'password'`, `'url'`, `'email'`, `'tel'` | `text` | The type of HTML input to render. |
 | `value` | `string` | `''` | The value of the input (used as a key/value pair in HTML forms with `name`). |
+| `assistiveText` | `string` | `''` | Allow the user to sets assistive text to display below the input element. |
```

Comment:
For other props when we refer to the user we mean the customer (who won't be setting the assistive text), so it's probably better to be a little more vague
```suggestion
| `assistiveText` | `string` | `''` | Allows assistive text to be displayed below the input element. |
```